internal/xds/translator/listener.go

```diff
@@ -106,6 +106,14 @@ func (t *Translator) addXdsHTTPFilterChain(xdsListener *listenerv3.Listener, irL
 		// merge adjacent slashes in the path
 		MergeSlashes:                 true,
 		PathWithEscapedSlashesAction: hcmv3.HttpConnectionManager_UNESCAPE_AND_REDIRECT,
+		Http2ProtocolOptions: &corev3.Http2ProtocolOptions{
```

Comment:
should this be set in L125 where we do specific things only for `isListener.IsHTTP2` ?

Reply:
Both should be fine I guess. This should only work with h2 anyways. But I could move it to the if conditional to prevent this config unnecessarily coming on the listener of non-h2 type.